apps/src/netsim/NetSimRemoteNodeSelectionPanel.js

```diff
@@ -169,7 +169,12 @@ NetSimRemoteNodeSelectionPanel.prototype.render = function () {
   NetSimUtils.makeContinueButton(this);
 
   this.addRouterButton_ = this.getBody().find('#netsim-lobby-add-router');
-  this.addRouterButton_.click(this.addRouterCallback_);
+  this.addRouterButton_.click(function () {
+    if (!this.addRouterButton_.is('[disabled]')) {
+      this.addRouterButton_.attr('disabled', true);
```

Comment:
Added a more robust lobby disable/re-enable control.  Cleans up another concern I've had for a while, with the "Cancel" button being exposed for a moment while connecting to a router (it's now disabled for the moment that it's visible).